### PR TITLE
Bump curl on Windows

### PR DESCRIPTION
**Pull request by cocreature**
The old url 404s now and upgrading seems nice anyway.

changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by aherrmann-da — APPROVED** [review]
LGTM

File: .dadew [hunk 1]
```diff
@@ -5,7 +5,7 @@
         "7zip",
         "bazel",
         "cacert",
-        "curl-7.65.3",
+        "curl-7.73.0",
```
Comment (SamirTalwar-DA) [inline .dadew:8]: Maybe just drop the version number from the file name?

**Review by SamirTalwar-DA — APPROVED** [review]

File: .dadew (hunk 1 above)
Comment (cocreature) [reply .dadew:8]: I think we should either strip it everywhere or add it everywhere. The current mixture is a mess. @aherrmann-da Is there an advantage to adding the suffix? Does it make upgrades more robust in some way?

Comment (aherrmann-da) [reply .dadew:8]: I'm not aware of an advantage that would be relevant to us. Dropping the suffix seems fine to me.

Comment (SamirTalwar-DA) [reply .dadew:8]: Great. Let's get this in and then we can delete all the version numbers later.

**Pull request merged**